### ISSUE TITLE
Display HTTP Code

### ISSUE THREAD

**Issue by JulioJu** [sharat87/roast.vim#6]
Hi !

It could be cool to also display the HTTP Code sent (400, 404, 500, etc.) by the server.

Thanks in advance ;-)

**Comment by sharat87** [comment]
Yes. I intend to have that display in the status bar of the response window. Coming soon :relaxed:.

**Comment by sharat87** [comment]
The status code should show up in the status line now.

**Comment by sharat87** [comment]
Yeah, I forgot to push that commit. It should be working fine now. Thanks.

**Comment by JulioJu** [comment]
Thanks :-)
